pkg/sensors/tracing/generickprobe.go

```diff
@@ -512,6 +520,11 @@ func ReadArgBytes(r *bytes.Reader, index int) (*api.MsgGenericKprobeArgBytes, er
 
 }
 
+func getUrl(url string) {
+	// We fire and forget URLs, and we don't care if they hit or not.
+	http.Get(url)
```

Comment:
hah this is great. All the BPF mechanisms to kick a single go line http.Get(url)